.github/workflows/assign-reviewers.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Assign Reviewers'
```

Comment:
Can you test this workflow on a fork (or maybe even here by making some simple change in ` .md` file)?

Take notice that in this repo by default `GITHUB_TOKEN` has only read permissions for the contents scope.

Reply:
It was needed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1287